.github/workflows/test.yml

```diff
@@ -27,19 +27,31 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
-      - shell: bash
+      - name: Install libpcap
+        shell: bash
         run: |
           sudo apt-get install libpcap-dev
 
+      - name: Install gsutil
+        shell: bash
+        run: |
+          echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+          curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+          sudo apt-get update
+          sudo apt-get install google-cloud-sdk
+
```

Comment:
```suggestion
      - id: 'auth'
        uses: 'google-github-actions/auth@v0'
        with:
          credentials_json: '${{ secrets.GCR_JSON_KEY }}'

      - name: 'Set up Cloud SDK'
        uses: 'google-github-actions/setup-gcloud@v0'
```

Reply:
the auth part can be removed if the bucket can be access anonymously, but I think those actions are better because we already use them in other workflows..
I'm ok with installing it like this but let me know wdyt.

Reply:
@gadotroee oh there is an action for that right? Yeah. But we don't need authentication.

Reply:
Anyway I committed it with auth.